src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,42 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  setSelectedTabId,
+  onTabSelected,
+}) => (
+  <div
+    data-cy="TabsComponent"
+  >
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            data-cy="Tab"
+            className={cn({ 'is-active':
+            (!tabs.find(item => (item.id === selectedTabId)) && tabs[0] === tab)
+            || (selectedTabId) === tab.id })}
```

Comment:
```suggestion
            (!tabs.find(item => item.id === selectedTabId)
```

Reply:
Good afternoon. I tried to apply this code, but unfortunately 2 tests don't pass. I optimized the condition as much as possible, but so that the tests pass:

(!tabs.some(item => item.id === selectedTabId) && tabs[0] === tab) || selectedTabId === tab.id

![image](https://github.com/mate-academy/react_tabs-js/assets/121498080/651ed685-b119-4de2-aca0-805da71b83eb)
![image](https://github.com/mate-academy/react_tabs-js/assets/121498080/b79b709b-830e-44b3-a1c0-ebb96cc501e8)